acceptance/analyzer_test.go

```diff
@@ -207,8 +231,8 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 
 		when("the provided layers directory isn't writeable", func() {
 			var imageName, authConfig string
-			var err error
 			it.Before(func() {
+				var err error
```

Comment:
I think it's better to move this line to the line before the usage of this variable, meaning, swap lines 235 and 236.

Reply:
Addressed in the following [commit](https://github.com/buildpacks/lifecycle/pull/646/commits/7b1d75feefd76dc76fb94799ed2756eeda7da14c)